### PR TITLE
Feature/pagtree clean

### PR DESCRIPTION
**Pull request by FinalAngel**
- Adds proper icons for apphooks, isHome and isSoftroot
- Fixes isSoftroot display (is now a separate entry)
- Adds proper icons and tooltip for info

image

image

### CODE REVIEW

**Comment by Brankub** [comment]
:+1:

File: cms/templates/admin/cms/page/tree/menu.html [hunk 1]
```diff
@@ -6,8 +6,18 @@
...
+                {% if page.soft_root %}
+                    <a href="{% url 'admin:cms_page_advanced' page.id %}">
+                        <span class="cms-icon cms-icon-sitemap" title="{% trans 'softroot' %}">
+                    </a>
+                {% endif %}
+                {% if page.application_urls %}
+                    <a href="{% url 'admin:cms_page_advanced' page.id %}">
+                        <span class="cms-icon cms-icon-puzzle" title="{% blocktrans with page.application_urls as apphook%}Application: {{ apphook }}{% endblocktrans %}">
+                    </a>
+               {% endif %}
```
Comment (dhorelik) [inline cms/templates/admin/cms/page/tree/menu.html]: indentation off here

**Comment by dhorelik** [comment]
overall lgtm

**Comment by mkoistinen** [comment]
LGTM

**Pull request merged**